### PR TITLE
Provide documentation and training for the ansible playbook and workflow

### LINKED ISSUE #102: Provide documentation and training for the ansible playbook and workflow
The implementation of the epic should provide documentation and diagrams to explain what has been configured

### PR DESCRIPTION
**Pull request by ThomasCardin**
- Closes #102

### CODE REVIEW

File: docs/ansible.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+## Executive Summary
+As part of our tasks, we aimed to create virtual machines (VMs) to facilitate the work of developers. The following needs were mentioned:
+
+- Testing applications before requesting IT to install the software on our personal laptops
+- Facilitating the work of certain developers (e.g., having CLIs already installed and configured)
+To set up virtual machines in the cloud, using Ansible was very useful to enable scaling in case we need more VMs in the future.
+
+## Glossary
+
+**Ansible:** Ansible is an open-source software tool for IT automation. It automates the provisioning, configuration, and deployment of servers and applications. Ansible is agentless, meaning it does not require any software to be installed on the machines it manages. It uses SSH to connect to machines and execute commands.
```
Comment (rngadam) [inline docs/ansible.md:12]: is there no line length checks? this is way >80 chars?

**Review by rngadam — CHANGES_REQUESTED** [review]

File: docs/ansible.md (hunk 1 above)
Comment (ThomasCardin) [reply docs/ansible.md:12]: Hmmmmmmm

Comment (ThomasCardin) [reply docs/ansible.md]: Done. About the markdown, I think jonathan is working on it inside this PR https://github.com/ai-cfia/github-workflows/pull/111

**Comment by SonOfLope** [comment]
I think .github needs to be added to folders excluded of ansible linting.

**Review by SonOfLope — APPROVED** [review]

**Review by rngadam — APPROVED** [review]

**Pull request merged**